.eslintrc.js

```diff
@@ -34,27 +34,19 @@ module.exports = {
 
   rules: {
     "babel/new-cap": "off",
-    "jsdoc/check-param-names": "warn",
-    "jsdoc/check-tag-names": "warn",
-    "jsdoc/check-types": "warn",
-    "jsdoc/newline-after-description": "warn",
-    "jsdoc/require-param": "warn",
-    "jsdoc/require-param-description": "warn",
-    "jsdoc/require-param-name": "warn",
-    "jsdoc/require-param-type": "warn",
-    "jsdoc/require-returns-description": "warn",
-    "jsdoc/require-returns-type": "warn",
+    "jsdoc/require-returns-description": "off",
+    "mozilla/balanced-listeners": "error",
+    "mozilla/no-aArgs": "warn",
+    "mozilla/use-chromeutils-import": "off", // TODO: "warn"?
 
     "comma-dangle": ["error", "always-multiline"],
     "eqeqeq": "error",
     "indent": ["warn", 2, {SwitchCase: 1}],
-    "mozilla/balanced-listeners": "error",
-    "mozilla/no-aArgs": "warn",
     "no-console": "warn",
-    "no-shadow": ["error"],
+    "no-shadow": "error",
     "no-unused-vars": "error",
-    "no-var": "off", // TODO: error
-    "no-warning-comments": ["off", {"location": "anywhere"}], // TODO: warn
+    "no-var": "error",
+    "no-warning-comments": ["warn", {"terms": ["todo", "fixme", /* TODO: add "needs_doc", */ "xxx"], "location": "anywhere"}],
```

Comment:
Excluding the "NEEDS_DOC" warnings for now until the code TODO comments are completed.